### ISSUE TITLE
Optical flow values get too large during training

### ISSUE THREAD

**Issue by heydarshahi** [RenYang-home/OpenDVC#23]
I changed the code to train the model with only end-to-end loss and from scratch (without the pre-trained optical flow network), and the optical flow values got very large as training progresses. I wonder if you have faced the same issue and how you could solve it. 

Of course, one workaround is to use multi-loss training. However, since the original DVC paper has done everything end-to-end, I think that's a problem with my code.  

Thanks!

Amin

--
PS. I changed the learning rate, and the problem still arises, but in later iterations.

**Comment by RenYang-home** [comment]
Hi,

I didn't meet this problem, and I also haven't tried training from scratch.

How large are your optical flow values?

**Comment by heydarshahi** [comment]
They can get arbitrarily large (up to 10^6) if I only use end-to-end loss. But when I use pretrained optical flow, they stay in the same range. I could reduce it by using regularization on the optical flow module. Still, I wondered if you've had the same problem. 

Thanks for the input!
